OneCommunityGlobal/HGNRest
# Nishita_team_stats_comparision_for_total_org_summary_page

# Description
This PR adds Team Stats comparison for users in teams and users with no teams 


## Related PRS (if any):
This PR is built off the backend PR https://github.com/OneCommunityGlobal/HGNRest/pull/1062.

## Main changes explained:
Updated the getTeamMembersCount method in overviewReportHelper.js to keep track of the users in team and users without teams and added the comparision stats for them.
…

## How to test:
1. Check into current branch
2. Do `npm install` and `...` to run this PR locally
3. Use Postman command to get an authorization token at http://localhost:4500/api/login using a JSON body with your developer account credentials:
{
    "email": "<YOUR EMAIL>"
    "password": "<YOUR PASSWORD>"
} 

4. Query the endpoint with Postman: api/reports/volunteerstats?startDate=2024-05-26&endDate=2024-06-02&comparisonStartDate=2024-05-18&comparisonEndDate=2024-05-25 using the above token
![image](https://github.com/user-attachments/assets/31d8ad6a-0eab-4b23-85bc-11d3f543129b)

5. Verify these highlighted fields are returned in the response:
![image](https://github.com/user-attachments/assets/fe8ca96e-0d52-40c6-8677-a2ffac2f7392)
